app/code/Magento/CatalogImportExport/Model/Import/Product/Validator/Media.php

```diff
@@ -17,7 +17,7 @@ class Media extends AbstractImportValidator implements RowValidatorInterface
      */
     const URL_REGEXP = '|^http(s)?://[a-z0-9-]+(.[a-z0-9-]+)*(:[0-9]+)?(/.*)?$|i';
 
-    const PATH_REGEXP = '#^(?!.*[\\/]\.{2}[\\/])(?!\.{2}[\\/])[-\w.\\/]+$#';
+    const PATH_REGEXP = '#^(?!.*[\\/]\.{2}[\\/])(?!\.{2}[\\/])[-\w.\s\\/]+$#';
```

Comment:
Please provide more details - when it may be needed? I remember only Windows guys wanted spaces somewhere to be allowed.

Reply:
On linux system whitespace are also allowed. I have that problem with a filename like "fg 1234.jpg". Before the import I already check the file path with file_exist method, there are no problems with the whitespaces. But this regex pattern said a whitespace is invalid and the import validation failed.

Reply:
Currently is the system asking you to rename file or now it works?